lib/rubocop/cop/rspec/verified_double_reference.rb

```diff
@@ -73,7 +73,7 @@ class VerifiedDoubleReference < Base
 
         def on_send(node)
           verified_double(node) do |class_reference|
-            return correct_style_detected unless opposing_style?(class_reference)
+            break correct_style_detected unless opposing_style?(class_reference)
```

Comment:
A bit of a hack to change the keyword because the line got too long 😄

Reply:
Even `next` would probably work, so there's a one-character space for improvement left 😆